fixtures/fiber-fragments/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
What's the advantage of having a fixture for this? Wouldn't this be well covered by the unit test?